File: DiaryNutrition/app/src/main/java/ediger/diarynutrition/data/Weight.java
package ediger.diarynutrition.data;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "weight")
public class Weight {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "_id")
    private int id;

    @ColumnInfo(name = "weight")
    private float amount;

    private long datetime;

    public Weight(float amount, long datetime) {
        this.amount = amount;
        this.datetime = datetime;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public float getAmount() {
        return amount;
    }

    public void setAmount(float amount) {
        this.amount = amount;
    }

    public long getDatetime() {
        return datetime;
    }

    public void setDatetime(long datetime) {
        this.datetime = datetime;
    }
}

File: app/src/main/java/ediger/diarynutrition/data/source/entities/Weight.kt
package ediger.diarynutrition.data.source.entities

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "weight")
class Weight(var amount: Float, var datetime: Long) {
    @PrimaryKey(autoGenerate = true)
    var id = 0
}